File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.java
// Last updated: 7/30/2025, 12:23:52 PM
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int max = 0;
        for(int i : nums){
            if(i==1){
                count ++;
                if(count > max){
                    max = count;
                }
            }
            else{
                count = 0;
            }
        }
        return max;
    }
}

File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.py
# Last updated: 7/30/2025, 12:23:56 PM
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        m = 0
        for i in nums:
            if i:
                count += 1
                if count > m:
                    m = count
            else :
                count = 0
        return m

                
        